libs/api/domains/regulations-admin/src/lib/graphql/regulationsAdmin.resolver.ts

```diff
@@ -96,4 +103,31 @@ export class RegulationsAdminResolver {
   async getDraftRegulationsLawChapters(@CurrentUser() { authorization }: User) {
     return await this.regulationsService.getRegulationsLawChapters(false)
   }
+
+  @Query(() => graphqlTypeJson)
+  async downloadRegulation(
```

Comment:
The name of this query is confusing to me. What does it actually do? Prepare the draft? Return a URL from the download service?

If so maybe it should be “prepareRegulationDownload”? And a mutation?

Also, please brush up on our [GraphQL Naming Conventions](https://docs.devland.is/technical-overview/api-design-guide/graphql-naming-conventions). We are one big happy family in GraphQL schema land, so everything needs to be namespaced:

```
prepareRegulationsAdmin[Regulation?]Download
```

Love it? 😊

Kind regards,
The GraphQL Schema Cop 👮‍♂️

Reply:
Feel free to address the namespace comment later to keep consistency in your domain schema and keep the scope of this PR small.